x/dex/types/params.go

```diff
@@ -18,6 +19,8 @@ var (
 	DefaultMaxJITsPerBlock       uint64 = 25
 	KeyGoodTilPurgeAllowance            = []byte("PurgeAllowance")
 	DefaultGoodTilPurgeAllowance uint64 = 540_000
+	KeyWhitelistedLPs                   = []byte("PurgeAllowance")
+	DefaultKeyWhitelistedLPs     []string
```

Comment:
maybe assign an empty array to default to be more clear here?

Reply:
linter does not like assigning an empty array